.ci/Jenkinsfile

```diff
@@ -137,21 +153,75 @@ pipeline {
       options { skipDefaultCheckout() }
       when {
         beforeAgent true
-        tag pattern: 'v\\d+\\.\\d+.*', comparator: 'REGEXP'
+        allOf {
+          //tag pattern: 'v\\d+\\.\\d+.*', comparator: 'REGEXP'
+          expression { env.CHANGE_ID ==~ /.*238/ }
+          expression { isInternalCI() }
+        }
       }
+      environment {
+        BUCKET_NAME = 'internal-ci-artifacts'
+        BUCKET_SUBFOLDER = "${env.REPO}/${env.TAG_NAME}"
+        BUCKET_PATH = "gs://${env.BUCKET_NAME}/${env.BUCKET_SUBFOLDER}"
+        BUCKET_CREDENTIALS = 'internal-ci-gcs-plugin'
+        SIGNED_ARTIFACTS = 'signed-artifacts'
+        BUCKET_SUBFOLDER_SIGNED_ARTIFACTS = "${env.BUCKET_SUBFOLDER}/${env.SIGNED_ARTIFACTS}"
+        BUCKET_SIGNED_ARTIFACTS_PATH = "gs://${env.BUCKET_NAME}/${env.BUCKET_SUBFOLDER_SIGNED_ARTIFACTS}"
+        RELEASE_URL_MESSAGE = "(<https://github.com/elastic/${env.REPO}/releases/tag/${env.TAG_NAME}|${env.TAG_NAME}>)"
+      }
+      // QUESTION: It's possible to be all the packages on Linux?
```

Comment:
@justinkambic  , @kyungeunni, is it possible to generate the distributable files (deb, rpm, windows, macos) on a Linux machine?

Reply:
Already answered somewhere else, sorry for the double question. Short answer, it's possible but it requires `wine`, so let's stick with `macos` for now.